lib/ansible/module_utils/profitbricks.py

```diff
@@ -0,0 +1,80 @@
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Comment:
code in module_utils must be BSD licensed

Reply:
@resmo It's fixed